include/nntile/starpu/add_fiber.hh

```diff
@@ -9,7 +9,8 @@
  *
  * @version 1.0.0
  * @author Aleksandr Mikhalev
- * @date 2023-07-21
+ * @author Aleksandr Mikhalev
+ * @date 2024-03-13
```

Comment:
main branch has updated header: it no longer contains authors and dates. But we keep all the rest: copyrights, short NNTile description, file path with file description and version